src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java

```diff
@@ -29,7 +29,7 @@
 import java.util.concurrent.TimeUnit;
 
 public class ViaVersionPlugin extends JavaPlugin implements ViaVersionAPI {
-
+    public static dobule version = 0.5.1;
```

Comment:
I don't used the method because on plugin.yml the version is "$version" and I don't know what it mean/work